### PR TITLE
New package: GiulioSorrentino.astroni version 1.0

### PR DESCRIPTION
**Pull request by GiulianoSpaghetti**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/111023&drop=dogfoodAlpha

### CODE REVIEW

File: manifests/g/GiulioSorrentino/astroni/1.0/GiulioSorrentino.astroni.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+PackageIdentifier: GiulioSorrentino.astroni
+PackageVersion: 1.0
+Installers:
+- InstallerUrl: http://numeronesoft.ddns.net//numeronesoft-backgrounds-otto-astroni.exe
+  Architecture: x64
+  InstallerType: portable
+  InstallerSha256: A60F5E75C238D914BDB10D28A99CF333A59CA1EA05C1DEA78A6F43C3B861849D
+  Commands:
+  - winrar
```
Comment (ItzLevvie) [inline manifests/g/GiulioSorrentino/astroni/1.0/GiulioSorrentino.astroni.installer.yaml:12]: Perhaps another word/term should be used here?

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Running the command also requires you to manually install the application:
image

Maybe this is an installer instead of a portable application.

**Comment by numerunix** [comment]
Installer type is not portable, is an exe created with winrar.
-s is silent install
-s1 is silent install with progress.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**